http-clients/src/main/java/com/palantir/remoting/http/UserAgents.java

```diff
@@ -30,8 +30,9 @@ public static String getUserAgent(String userAgent, String version) {
 
     /**
      * Constructs a user agent from the {@link java.util.jar.Attributes.Name#IMPLEMENTATION_TITLE} and {@link
-     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class. The default value for
-     * both properties is "{@value DEFAULT_VALUE}".
+     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class. Typically, the title
+     * and version are extracted from {@code MANIFEST.MF} entries of the Jar package containing the given class. The
+     * default value for * both properties is "{@value DEFAULT_VALUE}".
```

Comment:
thanks

On Mon, Jun 27, 2016 at 9:29 PM ellisjoe <notifications@github.com> wrote:

> In http-clients/src/main/java/com/palantir/remoting/http/UserAgents.java
> <https://github.com/palantir/http-remoting/pull/150#discussion_r68696062>:
>
> > @@ -30,8 +30,9 @@ public static String getUserAgent(String userAgent, String version) {
> >
> >      /**
> >       * Constructs a user agent from the {@link java.util.jar.Attributes.Name#IMPLEMENTATION_TITLE} and {@link
> > -     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class. The default value for
> > -     * both properties is "{@value DEFAULT_VALUE}".
> > +     * java.util.jar.Attributes.Name#IMPLEMENTATION_VERSION} of the package of the provided class. Typically, the title
> > +     * and version are extracted from {@code MANIFEST.MF} entries of the Jar package containing the given class. The
> > +     * default value for * both properties is "{@value DEFAULT_VALUE}".
>
> got a * between for and both
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/150/files/24db4fd214dadae5f8b35dfd1e9303c93fb5c287#r68696062>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGOdwS3tFH9OkNBYboUPt0C2XOi6mHPLks5qQKMTgaJpZM4I_uut>
> .
>